samples/XCT.Sample/App.xaml.cs

```diff
@@ -15,7 +14,6 @@ public App()
 
 			LocalizationResourceManager.Current.PropertyChanged += (sender, e) => AppResources.Culture = LocalizationResourceManager.Current.CurrentCulture;
 			LocalizationResourceManager.Current.Init(AppResources.ResourceManager);
-			LocalizationResourceManager.Current.CurrentCulture = new CultureInfo("en");
```

Comment:
This was added in nullable PR, and it's not necessary. `CurrentCulture` is nullable.